packages/mdc-select/mdc-select.scss

```diff
@@ -89,12 +83,110 @@
     z-index: 4; // Should pop up above everything else. temporary-drawer is next highest at 3.
   }
 
+  &__surface {
+    @include mdc-typography(subheading2);
+    @include mdc-theme-prop(color, text-primary-on-light);
+    @include mdc-rtl-reflexive-box(padding, right, $mdc-select-arrow-padding);
+    @include mdc-rtl-reflexive-box(padding, left, $mdc-select-label-padding);
+    @include mdc-ripple-surface;
+    @include mdc-ripple-color;
```

Comment:
Yeah, if anything I think they'd be folded into ripple or theme or something, I definitely don't think select should depend on text field. For now I'll just make select match text field.

Reply:
> As discussed offline, we are going to leave these ripple color values alone because they match the states work that Ken is doing. This means we can either make Text Field reflect this, or leave Text Field alone until states work commences. -amsheehan